diff -Naur '--exclude=*.pyc' old/backends/ssh_paramiko_backend.py now/backends/ssh_paramiko_backend.py
--- old/backends/ssh_paramiko_backend.py	2016-07-02 15:44:03.000000000 +0300
+++ now/backends/ssh_paramiko_backend.py	2016-08-23 18:31:24.014183400 +0300
@@ -406,8 +406,6 @@
 
         return sshconfig.lookup(host)
 
-duplicity.backend.register_backend("sftp", SSHParamikoBackend)
-duplicity.backend.register_backend("scp", SSHParamikoBackend)
 duplicity.backend.register_backend("paramiko+sftp", SSHParamikoBackend)
 duplicity.backend.register_backend("paramiko+scp", SSHParamikoBackend)
-duplicity.backend.uses_netloc.extend(['sftp', 'scp', 'paramiko+sftp', 'paramiko+scp'])
+duplicity.backend.uses_netloc.extend(['paramiko+sftp', 'paramiko+scp'])
diff -Naur '--exclude=*.pyc' old/backends/ssh_pexpect_backend.py now/backends/ssh_pexpect_backend.py
--- old/backends/ssh_pexpect_backend.py	2016-02-18 19:42:00.000000000 +0300
+++ now/backends/ssh_pexpect_backend.py	2016-08-23 18:31:15.527768500 +0300
@@ -287,6 +287,8 @@
         commandline = ("%s %s %s" % (self.sftp_command, globals.ssh_options, self.host_string))
         self.run_sftp_command(commandline, commands)
 
+duplicity.backend.register_backend("sftp", SSHPExpectBackend)
+duplicity.backend.register_backend("scp", SSHPExpectBackend)
 duplicity.backend.register_backend("pexpect+sftp", SSHPExpectBackend)
 duplicity.backend.register_backend("pexpect+scp", SSHPExpectBackend)
-duplicity.backend.uses_netloc.extend(['pexpect+sftp', 'pexpect+scp'])
+duplicity.backend.uses_netloc.extend(["sftp", "scp", 'pexpect+sftp', 'pexpect+scp'])
